src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php

```diff
@@ -47,6 +49,7 @@ trait ConfigurationTrait
     private $unaryMiddlewares;
     private $streamingMiddlewares;
     private $grpcInterceptors;
+    private $restHttpHandler;
```

Comment:
same as above - why not just call this `$httpHandler`?

Reply:
Done.